### PR TITLE
DGR-001: added models and postgreSQL dependency (temporarily)

### PR DESCRIPTION
**Pull request by kamilamie**

### CODE REVIEW

File: src/main/java/ru/itis/dogger/models/Answer.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```
Comment (AinaArd) [inline src/main/java/ru/itis/dogger/models/Answer.java]: Builder is anti pattern. Better use construstors

File: src/main/java/ru/itis/dogger/models/Dog.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```
Comment (AinaArd) [inline src/main/java/ru/itis/dogger/models/Dog.java]: Same about anti pattern

File: src/main/java/ru/itis/dogger/models/Meeting.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import java.util.Date;
+import java.util.List;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```
Comment (AinaArd) [inline src/main/java/ru/itis/dogger/models/Meeting.java]: Anti pattern

File: src/main/java/ru/itis/dogger/models/Message.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```
Comment (AinaArd) [inline src/main/java/ru/itis/dogger/models/Message.java]: Same as above

File: src/main/java/ru/itis/dogger/models/Owner.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.List;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```
Comment (AinaArd) [inline src/main/java/ru/itis/dogger/models/Owner.java]: Same as above

File: src/main/java/ru/itis/dogger/models/Place.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```
Comment (AinaArd) [inline src/main/java/ru/itis/dogger/models/Place.java]: Same as above

File: src/main/java/ru/itis/dogger/models/Question.java [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.List;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```
Comment (AinaArd) [inline src/main/java/ru/itis/dogger/models/Question.java]: Anti pattern

File: src/main/java/ru/itis/dogger/models/Answer.java [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+public class Answer {
+
+    @Id
+    @GeneratedValue
```
Comment (AinaArd) [inline src/main/java/ru/itis/dogger/models/Answer.java]: Where is generated strategy? Here and in other classes?

File: src/main/java/ru/itis/dogger/models/Question.java [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+public class Question {
+
+    @Id
+    @GeneratedValue
+    private Integer id;
```
Comment (AinaArd) [inline src/main/java/ru/itis/dogger/models/Question.java]: id is Integer?

File: src/main/java/ru/itis/dogger/models/WalkingArea.java [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```
Comment (AinaArd) [inline src/main/java/ru/itis/dogger/models/WalkingArea.java]: Anti pattern

File: src/main/resources/application.properties [hunk 11]
```diff
@@ -1 +1,11 @@
+spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=update
```
Comment (AinaArd) [inline src/main/resources/application.properties]: Bettern to use validate

File: src/main/java/ru/itis/dogger/models/Answer.java (hunk 1 above)
Comment (kamilamie) [reply src/main/java/ru/itis/dogger/models/Answer.java]: это Дима так говорит?

File: src/main/java/ru/itis/dogger/models/Question.java (hunk 9 above)
Comment (kamilamie) [reply src/main/java/ru/itis/dogger/models/Question.java]: ой

File: src/main/java/ru/itis/dogger/models/Answer.java (hunk 1 above)
Comment (AinaArd) [reply src/main/java/ru/itis/dogger/models/Answer.java]: лан, фиг с тобой

Comment (AinaArd) [reply src/main/java/ru/itis/dogger/models/Answer.java]: at least this reason:
There is no way to tell at compile time if all necessary fields are set.

Plain all-args constructor or factory method works better in this case - compiler will make sure that I'll pass all parameters necessary to create POJO. How correct are these parameters it's the another story, but all of them definitely will be there. With builder I can easily omit one or two. Issue will appear later, at run time, when I get NPE.

**Pull request merged**